### PR TITLE
222 readme.md refactoring

### PR DESCRIPTION
**Pull request by Blindspot22**
To enhance the clarity and organization of our project documentation, the following steps should be taken:

1. **Move Documentation**: Transfer  contents from the main `README.md` file into dedicated documentation folders (docs). This could include sections like usage examples, API references, and contribution guidelines. Each document should be clearly titled and organized in a way that makes it easy for users to navigate.

2. **Revise the README.md**: After relocating the documentation, focus on simplifying and clarifying the existing `README.md`. The updated version should provide a concise overview of the project, its purpose, and key features. Consider including:

   - A brief description of what the project does and why it's valuable.
   - Clear instructions on how to get started, including prerequisites and setup steps.
   - A short example of how to use the project effectively.
   - Links to the detailed documentation in the newly created folders for users who need more information.

3. **Ensure Readability**: Make the language straightforward and accessible, avoiding jargon unless necessary. Use bullet points, headings, and short paragraphs to break up the text, making it easier to read and understand.

By implementing these changes, the project documentation will be more structured and user-friendly, helping both new and experienced users quickly grasp the project's functionalities and navigate its resources.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
+
+Community Impact Guidelines were inspired by [Mozilla's code of conduct
+enforcement ladder](https://github.com/mozilla/diversity).
+
+[homepage]: https://www.contributor-covenant.org
+
+For answers to common questions about this code of conduct, see the FAQ at
+https://www.contributor-covenant.org/faq. Translations are available at
+https://www.contributor-covenant.org/translations.
```
Comment (Christiantyemele) [inline CODE_OF_CONDUCT.md:120]: @Blindspot22 are you the one to write the code of conduct ?

**Review by Christiantyemele — DISMISSED** [review]

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (Blindspot22) [reply CODE_OF_CONDUCT.md:120]: It's part of the README.md refinement as told by francis. And i wasn't the one who generated it. It is just a copy that is used for adorsys open source projects.

Comment (Christiantyemele) [reply CODE_OF_CONDUCT.md:120]: okay

File: README.md [hunk 2]
```diff
@@ -1,56 +1,15 @@
...
-
-## DIDComm Messaging
-
-**DIDComm** is a messaging protocol that works with DIDs to provide secure, private communication between parties. The full specification is available [here](https://identity.foundation/didcomm-messaging/spec/).
-
-### Functional Aspects
-
-* **Flexibility**: DIDComm messaging is designed to allow new protocols and applications to be built on top of it.
-* **Adoption**: Already adopted by several projects, including the W3C Verifiable Credentials Working Group.
+A DIDComm mediator is a cloud agent that facilitate secure, decentralized communication between mobile agents in the self-sovereign identity (SSI) ecosystem. This mediator acts as a cloud-based intermediary, enabling mobile agents to communicate without reliance on centralized infrastructures like Facebook, Signal, or Telegram.
```
Comment (IngridPuppet) [inline README.md:9]:
```suggestion
A DIDComm mediator is a cloud agent that facilitates secure, decentralized communication between mobile agents in the self-sovereign identity (SSI) ecosystem. This mediator acts as a cloud-based intermediary, enabling mobile agents to communicate without reliance on centralized infrastructures like Facebook, Signal, or Telegram.
```

**Review by IngridPuppet — DISMISSED** [review]

**Review by Tekum-Emmanuella — APPROVED** [review]

**Review by chendiblessing — APPROVED** [review]

**Pull request merged**